src/main/java/seedu/address/model/student/Attendance.java

```diff
@@ -19,12 +19,22 @@
 
     public final AttendanceEnum value;
 
+    /**
+     * Constructs a {@code Attendance}
+     *
+     * @param attendance An AttendanceEnum attendance
+     */
     public Attendance(AttendanceEnum attendance) {
```

Comment:
Would it be better to have a single constructor that takes in a string and validating from there, instead of having two separate constructors?

Reply:
hm yes. Would look into it for v2.0 thanks!

Reply:
No problem